packages/react-slider/src/components/Slider/useSliderStyles.ts

```diff
@@ -1,27 +1,185 @@
 import { makeStyles, mergeClasses } from '@fluentui/react-make-styles';
-// import { SliderState } from './Slider.types';
+import { HighContrastSelector } from '@fluentui/style-utilities';
```

Comment:
Let's comment the high contrast stuff for now and remove the dependency on `style-utilities` until we confirm this maybe in our next team sync, but I think we are moving towards using a high contrast theme instead of theming via the selector like we do now (more cross-browser and cross-platform support are some of the benefits of going this way iirc).